plugins/basic/menu.py

```diff
@@ -0,0 +1,39 @@
+import inspect
+import os
+import sys
+
+
+def plugin_startup():
+    """
+    Initializes the Toolkit plugin for Nuke.
+    """
+
+    current_file_path = os.path.abspath(
+        inspect.getsourcefile(lambda: 0)
+    )
+
+    plugin_root_path = os.path.dirname(current_file_path)
+
+    # the plugin python path will be just below the root level. add it to
+    # sys.path
+    plugin_python_path = os.path.join(plugin_root_path, "python")
+    sys.path.insert(0, plugin_python_path)
+
+    # now that the path is there, we can import the plugin bootstrap logic
+    try:
+        from tk_nuke_basic import plugin_bootstrap
+        plugin_bootstrap.bootstrap(plugin_root_path)
+    except Exception, e:
+        import traceback
+        stack_trace = traceback.format_exc()
+
+        message = "Shotgun Toolkit Error: %s" % (e,)
+        details = "Error stack trace:\n\n%s" % (stack_trace)
+
+        import nuke
+        nuke.error(message)
```

Comment:
Are we guaranteed to be in Nuke (not Nuke Studio or Hiero) when this is called? We'll let a NotImplementedError bubble up, otherwise.